redaxo/src/addons/media_manager/lib/effects/effect_header.php

```diff
@@ -10,6 +10,8 @@ public function execute()
         if ($this->params['cache'] == 'no_cache') {
             $this->media->setHeader('Cache-Control', 'must-revalidate, proxy-revalidate, private, no-cache, max-age=0');
             $this->media->setHeader('Expires', 'Sat, 26 Jul 1997 05:00:00 GMT'); // in the past
+        } elseif (isset($this->params['expires']) && 'no' !== $this->params['expires']) {
+            $this->media->setHeader('Expires', gmdate('D, d M Y H:i:s T', strtotime('+'.$this->params['expires'])));
```

Comment:
sollte man hier mit `Cache-Control: max-age` zusätzlich/stattdessn arbeiten?
soweit ich mich erinnern kann ist `Expires` ein "alter Hut".

Falls man `Cache-Control` verwendet, könnte man überlegen, ob  man auch noch `immutable` unterstützt

Reply:
Ehrlich gesagt, keine Ahnung, was hier sinnvoll ist.
Weißt du, ob Google sich bei `max-age` dann nicht mehr über "expiration not specified" beschwert? Oder du vielleicht @alexplusde?
Das war ja die Ursache für das Issue #572.

Reply:
google sollte mit Cache-Control zufrieden sein
https://stackoverflow.com/a/31351596/1597388

Reply:
`immutable` macht in diesem context übrigens doch keinen sinn... das setzt ja vorraus dass man urls hat die immer anders sind, was wir ja normalerweise nicht haben

Reply:
Ich denke, wir sollten dann doch nur "Cache-Control" nutzen, ich baue das um.
Und `immutable` finde ich schon gut. Wenn man selbst zum Beispiel einen Cache-Buster-Parameter an die Urls hängt, kann es sinnvoll sein, immutable zu nutzen.

Gerade bei den Core-Types könnten wir den Header-Effekt mit `immutable` hinzufügen, denn der MP fügt ja `&buster=...` an die URLs an.

Reply:
> denn der MP fügt ja &buster=... an die URLs an.

auch bei den bild-urls?

Reply:
Im Medienpool ja (zumindest in der Liste), in der Vorschau im Media(list)-Button nicht (da wird die Url per JS gebildert und man hat an der Stelle nicht die updatetime). Wobei letztere eigene MM-Typen haben, somit könnte man den Effekt dort weglassen.
So oder so, ist es aber nichts, was ich in diesem PR ändern möchte.